index.ts

```diff
@@ -410,11 +412,33 @@ export class GCP extends CloudPlatform<
                 }
             });
             // Troubleshooting data:
-            console.log(`Delete req ${JSON.stringify(deleteReq.data)}`);
+            console.log(`Delete EIP request ${JSON.stringify(deleteReq.data)}`);
+            // If the request has been queued check the VM for the current accessConfig
+            if(deleteReq.data.status !== 'SUCCEEDED'){
+                console.warn(`Delete has queued. Waiting for it to finish before adding Static IP`)
+                let running = true;
+                let startTime = Date.now();
+                const attachEIPTimeLimit = parseInt(ATTACH_EIP_TIME_LIMIT) || 45000 // Default 45 seconds max.
```

Comment:
can use a script level const DEFAULT_ATTACH_EIP_TIME_LIMIT = 4500 for this magic number.

Reply:
my comment refers to the 4500 in line 421.

Reply:
Sure, Either way works for me.

Reply:
No longer process.env now just:
ATTACH_EIP_TIME_LIMIT = 45000